src/test/resources/com/google/jenkins/plugins/computeengine/configuration-as-code.yml

```diff
@@ -0,0 +1,43 @@
+jenkins:
+  clouds:
+    - computeEngine:
+        cloudName: jenkins-build
+        projectId: gce-jenkins
+        instanceCapStr: 1
+        credentialsId: gce-jenkins
+        configurations:
+          - namePrefix:         jenkins-agent-image
+            description:        Jenkins agent
+            launchTimeoutSecondsStr: 6
+            retentionTimeMinutesStr: 300
+            mode:               EXCLUSIVE
+            labelString:        jenkins-agent
+            numExecutorsStr:    1
+            runAsUser:          jenkins
+            remoteFs:           '' # tried not setting this, field added when 'save' clicked in UI
+            windows:            false
+            windowsPasswordCredentialsId: ''    # tried not setting, added when saved in UI
+            windowsPrivateKeyCredentialsId: ''  # tried not setting, added when saved in UI
+            oneShot:            true
+            createSnapshot:     false
+            region:             "https://www.googleapis.com/compute/v1/projects/gce-jenkins/regions/europe-west1"
+            zone:               "https://www.googleapis.com/compute/v1/projects/gce-jenkins/zones/europe-west1-a"
+            template:           '' # tried not setting, added when 'saved' in UI
+            machineType:        "https://www.googleapis.com/compute/v1/projects/gce-jenkins/zones/europe-west1-a/machineTypes/n1-standard-2"
+            preemptible:        false
+            minCpuPlatform:     '' # tried not setting, added when 'saved' in UI
+            startupScript:      '' # tried not setting, added when 'saved' in UI
+            networkConfiguration:
+              sharedVpc:
+                projectId:      gce-jenkins-cloud-123456
```

Comment:
Suggest employing a placeholder here which can be text replaced in the test logic by the GOOGLE_PROJECT_ID env variable.

Reply:
What value it will give us?
Its only for one test which works like a charm with static values..

Reply:
Nvm, this is just used for unit tests. Disregard.